airbyte/metadata.csv

```diff
@@ -0,0 +1,52 @@
+airbyte.worker.activity.check_connection,count,,connection,,Increments when we start a check connection activity.,0,airbyte,activity check connection,
+airbyte.worker.activity.dbt_transformation,count,,,,Increments when we start a dbt transformation activity.,0,airbyte,activity dbt transformation,
+airbyte.worker.activity.discover_catalog,count,,,,Increments when we start a discover catalog activity.,0,airbyte,activity discover catalog,
+airbyte.worker.activity.failure,count,,,,Generic metric for incrementing when an activity fails. Add activity name to attributes.,0,airbyte,activity failure,
+airbyte.worker.activity.normalization,count,,,,Increments when we start a normalization activity.,0,airbyte,activity normalization,
+airbyte.worker.activity.normalization_summary_check,count,,,,Increments when we start a normalization summary check activity.,0,airbyte,activity normalization summary check,
+airbyte.worker.activity.refresh_schema,count,,,,Increments when we start a refresh schema activity.,0,airbyte,activity refresh schema,
+airbyte.worker.activity.replication,count,,,,Increments when we start a replication activity.,0,airbyte,activity replication,
+airbyte.worker.activity.spec,count,,,,Increments when we start a spec activity.,0,airbyte,activity spec,
+airbyte.worker.activity.submit_check_destination_connection,count,,connection,,Increments when we start a submit check connection activity.,0,airbyte,activity submit check destination connection,
+airbyte.worker.activity.submit_check_source_connection,count,,connection,,Increments when we start a submit check connection activity.,0,airbyte,activity submit check source connection,
+airbyte.worker.activity.webhook_operation,count,,,,Increments when we start a webhook operation activity.,0,airbyte,activity webhook operation,
+airbyte.worker.attempt.completed,count,,,,Increments when a new attempt is completed. One is emitted per attempt.,0,airbyte,attempt completed,
+airbyte.worker.attempt.created,count,,,,Increments when a new attempt is created. One is emitted per attempt.,0,airbyte,attempt created,
+airbyte.worker.attempt.created_by_release_stage,count,,,,Increments when a new attempt is created. Attempts are double counted as this is tagged by release stage.,0,airbyte,attempt created by release stage,
+airbyte.worker.attempt.failed_by_failure_origin,count,,,,"Increments for every failure origin a failed attempt has. Since a failure can have multiple origins, a single failure can be counted more than once. Tagged by failure origin and failure type.",0,airbyte,attempt failed by failure origin,
+airbyte.worker.attempt.failed_by_release_stage,count,,,,Increments when an attempt fails. Attempts are double counted as this is tagged by release stage.,0,airbyte,attempt failed by release stage,
+airbyte.worker.attempt.succeeded_by_release_stage,count,,,,Increments when an attempts succeeds. Attempts are double counted as this is tagged by release stage.,0,airbyte,attempt succeeded by release stage,
```

Comment:
```suggestion
airbyte.worker.attempt.completed,count,,attempt,,Increments when a new attempt is completed. One is emitted per attempt.,0,airbyte,attempt completed,
airbyte.worker.attempt.created,count,,attempt,,Increments when a new attempt is created. One is emitted per attempt.,0,airbyte,attempt created,
airbyte.worker.attempt.created_by_release_stage,count,,attempt,,Increments when a new attempt is created. Attempts are double counted as this is tagged by release stage.,0,airbyte,attempt created by release stage,
airbyte.worker.attempt.failed_by_failure_origin,count,,attempt,,"Increments for every failure origin a failed attempt has. Since a failure can have multiple origins, a single failure can be counted more than once. Tagged by failure origin and failure type.",0,airbyte,attempt failed by failure origin,
airbyte.worker.attempt.failed_by_release_stage,count,,attempt,,Increments when an attempt fails. Attempts are double counted as this is tagged by release stage.,0,airbyte,attempt failed by release stage,
airbyte.worker.attempt.succeeded_by_release_stage,count,,attempt,,Increments when an attempts succeeds. Attempts are double counted as this is tagged by release stage.,0,airbyte,attempt succeeded by release stage,
```